.gitignore

```diff
@@ -48,6 +48,9 @@ hs_err_pid*
 # Local environment
 .env
 
+# Mac OS
+.DS_Store 
+
 # Spring
 application-default.properties
```

Comment:
Could we either "wildcard" this to include `application-default.yml` or add another dedicated line for it ?